pangeo_forge_esgf/params.py

```diff
@@ -0,0 +1,27 @@
+# dataset id templates
+cmip6_template = "mip_era.activity_id.institution_id.source_id.experiment_id.variant_label.table_id.variable_id.grid_label.version"
+cordex_template = "project.product.domain.institute.driving_model.experiment.ensemble.rcm_name.rcm_version.time_frequency.variable"
```

Comment:
I dived a little more into the [ESGF API documentation](https://esgf.github.io/esg-search/ESGF_Search_RESTful_API.html) and added two functions to retrieve that information from the API now.

Reply:
Yes, the time frequency in CORDEX is described in "time_frequency", the values are not 100% percent compatible but they contain basically the same vocabulary as in CMIP5, e.g., "day", "mon", "1hr", etc... However, this is no indication of the time cell method, e.g., "1hrPt" as in CMIP6...

Reply:
I think we could also access the time frequency facet although it's not part of the dataset_id. It still has different names for the ESGF projects, (e.g. `time_frequency` in CORDEX but `frequency` in CMIP6). In general, the CORDEX vocabulary is derived from CMIP5 vocabulary. However, this could easily become a class attribute as you suggested...